LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/DefaultLoRaDataRequestHandler.cs

```diff
@@ -242,6 +244,27 @@ public async Task<LoRaDeviceRequestProcessResult> ProcessRequestAsync(LoRaReques
                                 requiresConfirmation = true;
                             }
                         }
+
+                        // Persist dwell time settings in device reported properties
+                        if (loraPayload.MacCommands is not null && loraPayload.MacCommands.Any(m => m.Cid == Cid.TxParamSetupCmd))
+                        {
+                            if (request.Region is DwellTimeLimitedRegion someRegion)
+                            {
+                                if (someRegion.DesiredDwellTimeSetting != loRaDevice.ReportedDwellTimeSetting)
+                                {
+                                    loRaDevice.UpdateDwellTimeSetting(someRegion.DesiredDwellTimeSetting, acceptChanges: false);
+                                    _ = await loRaDevice.SaveChangesAsync(force: true);
+                                }
+                                else
+                                {
+                                    this.logger.LogDebug("Received 'TxParamSetupAns' even though reported dwell time settings match desired dwell time settings.");
+                                }
+                            }
+                            else
+                            {
+                                throw new LoRaProcessingException($"Received 'TxParamSetupAns' in region '{request.Region.LoRaRegion}' which does not support dwell limitations.");
```

Comment:
Is this from the spec that this sis forbidden? otherwise I would simply log

Reply:
No I didn't find it anywhere in the spec - will do that and replace the exception with a log statement

Reply:
Done in 9cfe612c3fd6145065e3c07c01221603f1038bdf